### DISCUSSION TITLE
Polygon Mumbai priority fee always 50 gwei

### DISCUSSION THREAD

**Discussion by dimitriborgers3mint** [wevm/wagmi#1895] [General]
Hi all, two questions I'd love some help on!
1. Seems Polygon Mumbai priority fees are defaulted to 50 gwei. Where in the code base is this hard-coding taking place? It's supposed to call getFeeData from the provider, which in Alchemy's case would return 1.5 gwei every time, not 50.
2. Is prepare transaction hook really needed? Ethers and web3 will automatically fill in the gasLimit when sending a transaction, so what are we saving here?

**Comment by tmm** [comment]
1. wagmi doesn't hard-code fee values anywhere. Check ethers.
2. Depends if you want to avoid [these pitfalls](https://wagmi.sh/react/prepare-hooks#ux-pitfalls-without-prepare-hooks)

**Reply by dimitriborgers3mint** [reply]
@tmm Thanks for the response. Re: ethers, this is why I'm so confused. Ethers hardcodes to 1.5gwei...which should actually make the transaction fail, so no idea where this 50gwei is coming from
